scripts/package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "lectern-scripts",
```

Comment:
should this also be scoped within the org?
we oughtn’t pollute the registry for others